pkg/consts/clusterid.go

```diff
@@ -16,13 +16,13 @@ package consts
 
 const (
 	// ClusterIDConfigMapName is the name of the configmap where the cluster-id is stored.
-	ClusterIDConfigMapName = "cluster-id"
+	ClusterIDConfigMapName = "liqo-clusterid-configmap"
```

Comment:
Should not be relying on the name, since it is not guaranteed to remain that one.